### PR TITLE
Style low volume warning - TX Confirmation modal

### PR DESCRIPTION
**Pull request by W3stside**
Styles tx confirmation modal's low volume warning

Screenshot from 2020-08-10 17-41-51

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr1310--dexreact.review.gnosisdev.com

File: src/components/TradeWidget/TxMessage.tsx [hunk 1]
```diff
@@ -107,16 +107,20 @@ export const SimpleDisplayPrice: React.FC<SimpleDisplayPriceProps> = ({
...
+  padding: 1rem;
+  border-radius: 0.3em;
+
+  > span {
+    width: 94%;
+  }
 
   .alert {
-    min-width: 2.5em;
-    align-self: center;
```
Comment (Velenir) [inline src/components/TradeWidget/TxMessage.tsx:119]: Without these my alert :warning:  looks squished
pr1310--dexreact review gnosisdev com_trade_DAI-USDC_sell=1 price=1 00619 from= expires=

Comment (Velenir) [reply src/components/TradeWidget/TxMessage.tsx:119]: You can use `object-fit: contain;` if you don't like `align-self`

**Comment by W3stside** [comment]
Screenshot from 2020-08-12 10-47-25

**Pull request merged**